incubator/honeydipper/templates/daemon-deployment.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "honeydipper.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+    helm.sh/chart: {{ include "honeydipper.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.daemon.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "honeydipper.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      {{- with .Values.daemon.spec }}
+      {{ toYaml . | indent 6 | trim }}
+      {{- end }}
+      containers:
+        {{- if .Values.drivers.redis.local }}
+        - name: {{ .Chart.Name }}-redis
+          image: "redis:latest"
+        {{- end }}
+        - name: {{ .Chart.Name }}
+          {{- if .Values.daemon.useSource }}
```

Comment:
I find this very strange. Building from source on startup is IMO an anti-pattern which we should not promote here.

Reply:
Thanks @unguiculus, I will fix all these findings.  Yes, the building from source part was there before we established nightly build process.  I will take it out.